src/js/brandColor/brandColor.ts

```diff
@@ -37,7 +37,7 @@ export const brandColor: BrandColor = {
   orange300: '#faa66c',
   orange400: '#f8883b',
   orange500: '#f66a0a',
-  orange600: '#c65507',
+  orange600: '#bf5208',
```

Comment:
Thank god for tests. This fixes some wrong color values after https://github.com/MetaMask/design-tokens/pull/586 got merged and fixes failing test on main https://github.com/MetaMask/design-tokens/actions/runs/7644751686/job/20829744608